src/index.ts

```diff
@@ -279,33 +285,34 @@ export default class DeviseTokenAuthClient {
     constructor(params: Partial<Config> | Array<Record<string, Partial<Config>>>) {
         // user is using multiple concurrent configs (>1 user types).
         if (params instanceof Array && params.length) {
-            // extend each item in array from default settings
-            for (let i = 0; i < params.length; i++) {
-                // get the name of the config
-                const conf = params[i];
-                let label = '';
-                // eslint-disable-next-line no-restricted-syntax,guard-for-in
-                for (const k in conf) {
-                    label = k;
-
-                    // set the first item in array as default config
-                    if (i === 0) {
-                        defaultConfigName = label;
-                    }
-                }
-
-                // use copy preserve the original default settings object while
-                // extending each config object
-                const defaults = cloneDeep(configs.default);
-                const fullConfig = {} as any;
-                fullConfig[label] = Object.assign(defaults, conf[label]);
-                Object.assign(configs, fullConfig);
-            }
-
-            // remove existing default config
-            if (defaultConfigName !== 'default') {
-                delete configs.default;
-            }
+            throw new Error('DeviceTokenAuthClient config as an Array is not supported.\nWe need to migrate cloneDeep from lodash/fp/cloneDeepDeep first.');
```

Comment:
Small fix for the "cloneDeepDeep"

```suggestion
            throw new Error('DeviceTokenAuthClient config as an Array is not supported.\nWe need to migrate cloneDeep from lodash/fp/cloneDeep first.');
```